scripts/build_provenance

```diff
@@ -10,7 +10,11 @@ readonly OUTPUT_PATH=$2
 # Shift arguments so that the remaining arguments are the just build arguments.
 shift 2
 
-./scripts/docker_run "${@}"
+# Run the build in a simplified docker configuration that uses the default
```

Comment:
What is the default docker user? Are the builds reproducible if we use the default docker user?

Reply:
It's root. :) But also whichever the default docker user is, it'll be the same across here, and the transparent release CLI. 

And yeah, the provenance produced by this is successfully verified by the transparent release CLI, tested by running thie provenance GitHub action (that runs this script) on my Oak fork

Reply:
Also, once we add kokoro to verify the generated provenances (soon, CL is ready) we'll fail GitHub checks if ever the build stops being reproducible.

Reply:
Is that a post-merge check?

Reply:
atm it would be, but we can decide to make it pre merge if using CI runner resources for it is fine :) It probs would be

Reply:
Thanks. I think post-merge is fine for now. I wonder how it shows up on the CI, since it is triggered by the changes in the provenance branch, I think.

Reply:
Yup atm it would be. It'd show the same way checks show on the main branch, as a checkmark or cross icon next to each commit. Down the road we should increase the visibility of that check failing